README.md

```diff
@@ -13,3 +13,25 @@ It is meant to be run regularly by a cronjob.
 ```bash
 paster --plugin=ckanext-stadtzh-theme stadtzhtheme cleanup_datastore -c /etc/ckan/default/development.ini
 ```
+
+## Logic for Autosuggestion
+
+This extensions currently provides one action for collect autosuggestion
```

Comment:
extension (singular)
... currently provides one action to collect autosuggestions from ...